docs/intro-to-coalton.md

```diff
@@ -175,8 +178,9 @@ Here is an example of using a curried function to transform a list.
   (map (+ 2) nums)) ;; 4 5 6 7
 ```
 
-There are convenient *syntaxes* for composing functions with the
-`pipe` and `nest` macros.
+### Pipelining Syntax and Function Composition
+
+There are convenient *syntaxes* for composing functions with the `pipe` and `nest` macros.
```

Comment:
Why is "*syntaxes*" italicized?

Reply:
well, for emphasis of course. they're not functions for composing functions :)